sled-agent/src/rack_setup/service.rs

```diff
@@ -281,14 +282,31 @@ impl ServiceInner {
 
         info!(self.log, "sending service requests...");
         let services_put = || async {
+            let mut generation = Generation::new();
```

Comment:
During rack setup, we don't really care about the generation numbers, so we basically just keep bumping this until it works.